android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -1,18 +1,21 @@
 package com.swmansion.gesturehandler.react
 
 import android.content.Context
+import android.util.Log
```

Comment:
remove

Reply:
Removed in https://github.com/software-mansion/react-native-gesture-handler/pull/1670/commits/6dcc8e79a56e8fb46c1ea1c3cef46d42c0648944